### ISSUE TITLE
Switch: Possible improvements to the UE (low priority)

### ISSUE THREAD

**Issue by Instandhaltung** [mgba-emu/mgba#1560]
Hey nice people making mgba,
since I am stupid and I don't know where to put it, here are some features I would love to see in the future for the standalone build on Switch (biggest wishes on top). Maybe some could be implemented to ease and improve the usability of the emulator:

**1. Map a key to multiple buttons/ Enhanced custom mapping.** For example, it would be nice to be able to map the GBA R button to R _and_ ZR on Switch, or the A button to A _and_ Y or something. 
**2. Add custom options for individual games.** Since there is a new renderer in mGBA, it might be useful to be able to apply it to individual games, since it works perfectly for some games, but stutters on others. The more options the emulator has, the more useful specific options for individual games might become.
**3. Customize the rumble intensity.** Some games rumble quite a lot (I look at you, Pokemon Pinball for GBA!), so it might be cool to reduce the rumble to a low setting.
**4. Being able to use the GBC bios for GB games.** This would allow to customize the palette of those games like on the GBC. I know there is something being worked on regarding GBC palettes, but I can't see it yet.
**5. Use SGB borders that are still implemented in early GBC games.** Some early GBC games (like Pokemon Gold and Silver) still have SGB borders. It might be cool to check for them, even if the main game is in GBC mode. GameYob on NDS did that, so it might be possible here as well.

I know these options are just "nice to have", but I wanted to voice them nontheless.
Thanks for your hard work!

**Comment by Instandhaltung** [comment]
I know bumping is stupid, but have you thought about mapping a key to multiple buttons on Switch? Might be very helpful!

**Comment by endrift** [comment]
Yes, that's going to be coming at the same time as the other ports after I'm done with the input revamp.

**Comment by eadmaster** [comment]
added some hotkeys [here](https://github.com/eadmaster/mgba-libretro/releases/tag/test) and remapped L->ZL, R->ZR
